config.js

```diff
@@ -822,6 +822,8 @@ config.NSFS_UPDATE_ISSUES_REPORT_ENABLED = true;
 config.NSFS_EXIT_EVENTS_TIME_FRAME_MIN = 24 * 60; // per day
 config.NSFS_MAX_EXIT_EVENTS_PER_TIME_FRAME = 10; // allow max 10 failed forks per day
 
+process.env.NSFS_ENABLE_DINAMIC_SUPPLEMENTAL_GROUPS = 'true';
```

Comment:
```suggestion
process.env.NSFS_ENABLE_DYNAMIC_SUPPLEMENTAL_GROUPS = 'true';
```

Reply:
can you set it to be a regular config? we don't set here envs

Reply:
yes, from what I could see we currently, only override env, when there is a configuration that overrides it. I can add a section that convert default configurations as well. also I saw that there is an env file. but looks like its used only for GPFS

Reply:
hmm not only for GPFS, currently a custom config.json is used by NC
but I think that default should work with these lines - 
``` 
       Object.keys(merged_config).forEach(function(key) {
            const config_to_env = ['NOOBAA_LOG_LEVEL', 'UV_THREADPOOL_SIZE', 'GPFS_DL_PATH'];
            if (config_to_env.includes(key)) {
                process.env[key] = merged_config[key];
                return;
            }
            config[key] = merged_config[key];
        });
```
can you please check?